remix.init/index.js

```diff
@@ -56,18 +64,45 @@ const getPackageManagerVersion = (packageManager) =>
 
 const getRandomString = (length) => crypto.randomBytes(length).toString("hex");
 
-const updatePackageJson = ({ APP_NAME, packageJson }) => {
+const updatePackageJson = ({ APP_NAME, packageJson, packageManager }) => {
   const {
     scripts: {
       // eslint-disable-next-line no-unused-vars
       "format:repo": _repoFormatScript,
       ...scripts
     },
+    devDependencies: { ...devDependencies },
+    prisma: { ...prisma },
   } = packageJson.content;
 
+  if (packageManager.tsNative) {
+    delete devDependencies["ts-node"];
+  }
+
+  const updateScripts = (scripts) => {
```

Comment:
there are a lot of npm-specific commands in the scripts, so this tries to replace them all with the correct one. This should effectively be a noop if using `npm`.

e.g. `npx ts-node` => `bun`, `npm run` => `yarn`, etc.